free-1.md

```diff
@@ -57,12 +116,37 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I'd say maybe 20-30% of their time would be spent on engaging in the language aspects. Some questions that remain to be answered:
+
+* Should code structures be opt-in or opt-out in the flowchart?
+..* Based on this decision, what does it look like to opt in or out, say, an `if` statement?
+* What language should be used to generate the flowchart?
+..* What effect do potential languages have on the processing logic?
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think the idea has potential to grow, but I think one could successfully see multiple meaningful iterations
+before the semester's end.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+**Benefits**
+* Well-scoped
+* Have to consider adapting the proposed DSL to the language for which it's designed (considering both this language's syntax and style)
+* Have to consider complexity of flowchart creation language
+* Potential to extend or modify an existing language
+
+**Drawbacks**
+* There is an existing, though incomplete, language that addresses the need fairly well.
+
+
+[Javadoc]: https://en.wikipedia.org/wiki/Javadoc
+[Doxygen]: https://en.wikipedia.org/wiki/Doxygen
+[code2flow]: http://code2flow.com/
+[Flowgen]: http://jlopezvi.github.io/Flowgen/index.html
+[Flowgen Paper]: http://arxiv.org/pdf/1405.3240.pdf
```

Comment:
nice, i definitely got more and more sold on this idea as I read more. Sorry for any irrelevant comments!